src/notebooks/controllers/commands/installPythonControllerCommands.ts

```diff
@@ -177,8 +171,7 @@ export class InstallPythonControllerCommands implements IExtensionSingleActivati
             // Now start to indicate that we are performing the install and locating kernels
             const reporter = this.progressReporter.createProgressIndicator(DataScience.installingPythonExtension());
             try {
-                // Directly install the python extension
-                await this.commandManager.executeCommand('workbench.extensions.installExtension', PythonExtension);
+                await this.extensionChecker.directlyInstallPythonExtension();
```

Comment:
I had added a helper function before and forgot to use it, so swapped it in now.